cmd/configure.go

```diff
@@ -168,11 +176,11 @@ func saveConfiguration(cmd *cobra.Command) error {
 
 	if rootConfig.Token == "" || overwrite {
 		fmt.Println("Please enter the APISIX token: ")
-		token, err := reader.ReadString('\n')
+		byteToken, err := term.ReadPassword(0)
```

Comment:
1. if is terminal, use `syscall.Stdin` instead of 0
2. if not, handle tty is necessary

Reply:
Changed to use `syscall.Stdin`.